server/src/main/java/org/eclipse/openvsx/ExtensionValidator.java

```diff
@@ -55,6 +59,47 @@ public Optional<Issue> validateNamespace(String namespace) {
         return Optional.empty();
     }
 
+    public List<Issue> validateNamespaceDetails(NamespaceDetailsJson json) {
+        var issues = new ArrayList<Issue>();
+        checkCharacters(json.displayName, "displayName", issues);
+        checkFieldSize(json.displayName, 32, "displayName", issues);
+        checkCharacters(json.description, "description", issues);
+        checkFieldSize(json.description, DEFAULT_STRING_SIZE, "description", issues);
+        checkURL(json.companyLink, "companyLink", issues);
```

Comment:
idea (low-priority): I saw that the MS marketplace also calls it the company link, but for personal namespaces, do you think that maybe this can be just "website"? This would cover both cases and make everyone happy

Reply:
Makes sense, I'll rename it to website.